app/translations/i18n.js

```diff
@@ -0,0 +1,32 @@
+const path = require('path');
+const electron = require('electron');
+const fs = require('fs');
+
+const ConfigUtil = require(__dirname + '/../renderer/js/utils/config-util.js');
+
+class i18n {
+	constructor() {
+		this.app = electron.app ? electron.app : electron.remote.app;
+		this.appLanguage = ConfigUtil.getConfigItem('spellcheckerLanguage');
+		this.appLocale = path.join(__dirname, this.appLanguage + '.json');
+
+		if (fs.existsSync(this.appLocale)) {
+			this.loadedLanguage = JSON.parse(fs.readFileSync(this.appLocale, 'utf8'));
```

Comment:
Right here if we add a check for `isDev` and if not in development we `require` the json it would be cached by node and optimized by electron which will be faster.

cc @akashnimare